src/harness/testcases/testdata/fss_record_0.json

```diff
@@ -8,18 +8,21 @@
         "latitude": 39.2291,
         "longitude":  -100.1,
         "height": 7.6,
+        "elevation": 100.0,
         "heightType": "AGL",
         "antennaGain": 16,
         "antennaBeamwidth": 30,
         "antennaAzimuth": 270,
-        "antennaDowntilt": -90
+        "antennaDowntilt": -90,
+        "antennaElevationAngle":39
       },
       "operationParam": {
         "operationFrequencyRange": {
           "lowFrequency": 3650000000,
           "highFrequency": 4200000000
         }
-      }
+      },
+      "ttc": true
```

Comment:
I think this needs to be moved to a higher level since we cannot change the IncumbentProtectData object from SAS-SAS v1.0. Please also make sure that the other fields use names that are consistent with the definition of that object, or are defined outside the object.